Assets/MixedRealityToolkit.Providers/ObjectMeshObserver/SpatialObjectMeshObserver.cs

```diff
@@ -130,30 +132,6 @@ public override void Reset()
             Initialize();
         }
 
-        /// <inheritdoc />
-        public override void Enable()
-        {
-            // Resume iff we are not running and had been disabled while running.
-            if (!IsRunning && autoResume)
```

Comment:
Interestingly, this doesn't match the pasted code in https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6908 and also seems like it should resolve to `false` on first start? Since `autoResume` defaults to `false`.

Reply:
`autoResume` is missing in the reported issue

Reply:
Oh I see this isn't the WMR class. Whoops!